src/libImaging/JpegEncode.c

```diff
@@ -137,6 +137,20 @@ ImagingJpegEncode(Imaging im, ImagingCodecState state, UINT8 *buf, int bytes) {
             /* Compressor configuration */
             jpeg_set_defaults(&context->cinfo);
 
+            /* Prevent RGB -> YCbCr conversion */
+            if (context->keep_rgb) {
+                switch (context->cinfo.in_color_space) {
+                case JCS_RGB:
+#ifdef JCS_EXTENSIONS
+                case JCS_EXT_RGBX:
+#endif
+                    jpeg_set_colorspace(&context->cinfo, JCS_RGB);
+                    break;
+                default:
+                    break;
```

Comment:
```suggestion
                    case JCS_RGB:
#ifdef JCS_EXTENSIONS
                    case JCS_EXT_RGBX:
#endif
                        jpeg_set_colorspace(&context->cinfo, JCS_RGB);
```
A stylistic suggestion. Other people might have different ideas, so feel free to take or leave this as you see fit.

Reply:
I've redone the indent.  The other stuff is there to prevent compiler warnings about missing switch cases (since the switch expression is an enum) and fallthrough.  I don't know what compiler flags Pillow uses under all circumstances, but it seemed better to be explicit.

Reply:
I don't think it would generate warnings, but I do think it is easier to read with the `break`s and `default`.
Fall-through is intentionally used in `_imagingft.c`, but it is accompanied by a comment to make it clear it is intentional: https://github.com/python-pillow/Pillow/blob/f9c7bd86115090ce2b110794485f138f0d5d5953/src/_imagingft.c#L1023-L1025